vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/converter/ParametersToSequenceTouchActionConverter.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.mobileapp.converter;
+
+import java.lang.reflect.Type;
+import java.util.function.Supplier;
+
+import javax.inject.Named;
+
+import org.jbehave.core.steps.ParameterConverters.AbstractParameterConverter;
+import org.jbehave.core.steps.Parameters;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+import org.vividus.bdd.converter.ui.PointConverter;
+import org.vividus.bdd.converter.ui.StringToLocatorConverter;
+import org.vividus.mobileapp.action.SequenceTouchAction;
+import org.vividus.mobileapp.action.SequenceTouchActionType;
+
+@Named
+public class ParametersToSequenceTouchActionConverter
```

Comment:
is it possible to reuse org.vividus.bdd.converter.ui.web.ParametersToSequenceActionConverter? this class and that one are identical in terms of logic, maybe we can introduce some level of abstraction of hide coupling to exact types